tests/dataplane-performance/results/1.1.0/1.1.0.md

```diff
@@ -0,0 +1,123 @@
+# Results
+
+## Test environment
+
+GKE cluster:
+
+- Node count: 3
+- Instance Type: e2-medium
+- k8s version: 1.27.3-gke.100
+- Zone: europe-west2-b
+- Total vCPUs: 6
+- Total RAM: 12GB
+- Max pods per node: 110
+
+Test VM:
+
+- Instance Type: e2-medium
+- Zone: europe-west2-b
+- vCPUS: 2
+- RAM: 4GB
+
+NGF deployment:
+
+- NGF version: edge - git commit 21a2507d3d25ac0428384dce2c042799ed28b856
+- NGINX Version: 1.25.3
+
+## Summary
+
+- Path based routing performance has decreased slightly since 1.0.0
```

Comment:
This should just relate to nginx right? And all that was changed was the patch version bump.